docs/workingWithLostromos.md

```diff
@@ -29,6 +29,7 @@ To start working with Lostrómos, you should begin by playing around with some b
   * `group` (Required) The group of the CRD you want monitored (ex: stable.wpengine.io)
   * `version` (Required) The version of the CRD you want monitored
   * `namespace` The namespace of the CRD you want monitored
+  * `filter` Filter to specify if Lostromos will act on a resource create/update/delete
```

Comment:
Probably should include a link to the events doc here for help understanding how this works.

Reply:
Done